### DISCUSSION TITLE
Ability to click weather to see weekly forecast

### DISCUSSION THREAD

**Discussion by Orig-Technofuzz** [OpenRCT2/OpenRCT2#20768] [Feature requests]
I think it would be a nice addition to be able to see the weekly forecast for the weather.  It would be even funnier if the outcome of the weather forecast was not 100% accurate.  Sometimes i just have ideas and i'm not sure if they're good ones.  Please let me know what everyone thinks of this.

**Comment by duncanspumpkin** [comment]
Future weather is decided at random so it would be slightly annoying to do this. You would have to store more state for the weather (or you could just store the weather as a random number seed). Either way it requires modifying the save format for not much benefit.
